examples/agn_example.py

```diff
@@ -26,9 +27,9 @@
 broad = 30.
 
 # Initialize spectrum object
-spec = pyspeckit.Spectrum('../tests/sample_sdss.txt')
-spec.units = 'erg s^{-1} cm^{-2} \\AA^{-1}'
-spec.xarr.units='angstroms'
+spec = pyspeckit.Spectrum('sample_sdss.txt')
+spec.unit = 'erg s^{-1} cm^{-2} \\AA^{-1}'
+spec.xarr.set_unit = u.dimensionless_unscaled
```

Comment:
`set_unit` is a method.  You should be setting the unit to `angstroms` (or `u.AA`) here anyway:
`spec.xarr.set_unit(u.AA)`